blockchain/types/src/block_metadata.rs

```diff
@@ -22,9 +47,10 @@ pub struct BlockMetadataContents {
     #[prost(message, required, tag = 2)]
     quorum_set: QuorumSet,
 
-    /// IAS report for the enclave which generated the signature.
-    #[prost(message, required, tag = 3)]
-    verification_report: VerificationReport,
+    /// Attestation evidence for the enclave which generated the signature.
+    #[prost(oneof = "AttestationEvidence", tags = "3, 5")]
+    #[digestible(name = "verification_report")]
+    attestation_evidence: Option<AttestationEvidence>,
```

Comment:
The use of `Option` here is similar to the comment on #3555 on questioning if we should be using prost up here at this level. This really isn't optional, there should always be one of the variants, but to work with prost this has to be an option.

Reply:
I'm not sure its similar since this is a oneof field...

Reply:
true, it's a prost limitation in that there is no default enum variant when there isn't a simple enum, so to get a default, `Option<T>` was used.